Content.Shared/Teleportation/Systems/SwapTeleporterSystem.cs

```diff
@@ -184,20 +194,20 @@ public void DestroyLink(Entity<SwapTeleporterComponent?> ent, EntityUid? user)
             DestroyLink(linked, user); // the linked one is shown globally
     }
 
-    private EntityUid GetTeleportingEntity(Entity<TransformComponent> ent)
+    private (EntityUid, BaseContainer?) GetTeleportingEntity(Entity<TransformComponent> ent)
     {
         var parent = ent.Comp.ParentUid;
         if (_container.TryGetOuterContainer(ent, ent, out var container))
             parent = container.Owner;
 
         if (HasComp<MapGridComponent>(parent) || HasComp<MapComponent>(parent))
-            return ent;
+            return (ent, container);
 
         if (!_xformQuery.TryGetComponent(parent, out var parentXform) || parentXform.Anchored)
-            return ent;
+            return (ent, container);
 
         if (!TryComp<PhysicsComponent>(parent, out var body) || body.BodyType == BodyType.Static)
-            return ent;
+            return (ent, container);
 
         return GetTeleportingEntity((parent, parentXform));
     }
```

Comment:
This looks like it doesn't check caninsert anymore?

Reply:
CanInsert does not do a check that would prevent a player from getting teleported into a microwave or another player's hand. 
![image](https://github.com/space-wizards/space-station-14/assets/83733158/f9cbbae4-3bca-4b59-8d22-c88d75ecd254)

https://github.com/space-wizards/space-station-14/assets/83733158/86110f4b-c2ee-4b4d-88d6-6f536f3c757d